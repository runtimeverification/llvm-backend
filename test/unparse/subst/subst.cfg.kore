// RUN: %kprint-check
/* Spc */
\or{SortGeneratedTopCell{}}(
    /* Created: Kore.Internal.OrPattern.toTermLike */
    /* Spa */
    \and{SortGeneratedTopCell{}}(
        /* Fl Fn D Sfa Cl */
        Lbl'-LT-'generatedTop'-GT-'{}(
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'k'-GT-'{}(
                /* Fl Fn D Sfa Cl */
                kseq{}(
                    /* Fl Fn D Sfa Cl */ Lblone'Unds'TEST'Unds'KItem{}(),
                    /* Fl Fn D Sfa Cl */ dotk{}()
                )
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'generatedCounter'-GT-'{}(
                /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
            )
        ),
        /* Spa */
        \equals{SortInt{}, SortGeneratedTopCell{}}(
            /* Fl Fn D Sfa */ Var'QuesUnds'0:SortInt{},
            /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("1")
        )
    ),
    /* Created: Kore.Internal.OrPattern.toTermLike */
    /* Spc */
    \and{SortGeneratedTopCell{}}(
        /* Fl Fn D Sfc */
        Lbl'-LT-'generatedTop'-GT-'{}(
            /* Fl Fn D Sfc */
            Lbl'-LT-'k'-GT-'{}(
                /* Fl Fn D Sfc */
                kseq{}(
                    /* Fl Fn D Sfc */
                    /* Inj: */ inj{SortInt{}, SortKItem{}}(
                        /* Fl Fn D Sfa */ Var'QuesUnds'0:SortInt{}
                    ),
                    /* Fl Fn D Sfa Cl */ dotk{}()
                )
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'generatedCounter'-GT-'{}(
                /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
            )
        ),
        /* Sfa */
        \not{SortGeneratedTopCell{}}(
            /* Spa */
            \equals{SortInt{}, SortGeneratedTopCell{}}(
                /* Fl Fn D Sfa */ Var'QuesUnds'0:SortInt{},
                /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("1")
            )
        )
    )
)
