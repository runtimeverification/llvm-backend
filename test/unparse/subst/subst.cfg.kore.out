    #Not ( {
      ?_0:Int
    #Equals
      1
    } )
  #And
    <k>[0m
      ?_0:Int ~> .
    </k>[0m
#Or
  <k>[0m
    one[0m ~> .
  </k>[0m
